src/components/Sections/SectionList/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+import './index.scss';
+
+const SectionList = ({
+  name,
+  description,
+  keywords,
+  url,
+  roles,
+  entity,
+  type,
+  startDate,
+  endDate,
+  highlights,
+  position,
+  summary,
+  organization,
+  sectionName,
+  reference,
+  language,
+  fluency,
+}) => {
+  const _highlights = highlights?.map((data, index) => {
+    return <li key={index} dangerouslySetInnerHTML={{ __html: data }}></li>;
+  });
+
+  const _keywords = keywords?.map((data, index) => {
+    return <li key={index} dangerouslySetInnerHTML={{ __html: data }}></li>;
+  });
+
+  const _roles = roles?.map((data, index) => {
+    return <li key={index} dangerouslySetInnerHTML={{ __html: data }}></li>;
+  });
+  if (
+    sectionName == 'Work Experience' ||
+    sectionName == 'Volunteer Work' ||
+    sectionName == 'Projects'
+  ) {
+    return (
+      <div className="section-list">
+        <div className="section-list__info">
+          <div>
+            <div className="section-list__name">{name || organization}</div>
+            <div className="section-list__role">{_roles || position}</div>
+          </div>
+          <div className="section-list__date">
+            {startDate}-{endDate}
```

Comment:
Try to add formatter here. You can use this library: https://date-fns.org

Otherwise currently it looks weird
<img width="923" alt="Screenshot 2021-09-19 at 1 39 54 PM" src="https://user-images.githubusercontent.com/29898106/133920311-2f2bbe24-ed9b-4775-8242-8a8cc2017b4e.png">


It should have same date convention through out the resume something like this:
<img width="192" alt="Screenshot 2021-09-19 at 1 39 44 PM" src="https://user-images.githubusercontent.com/29898106/133920330-a6ab2b13-cd3a-4083-9335-41a90fdbddf5.png">